### PR TITLE
Release notes

### LINKED ISSUE #163: "What's new" section
Show the latest changes in a box on the unlock page, if the user hasn't seen them for the latest version. Fetch the changelog and the changes for the latest version.

### LINKED ISSUE #218: some input fields are not recognized by buttercup
I don't know why this does happen but there are some pages with login forms not recognized correctly by the buttercup addon. There is no buttercup icon next to the input field for the login name.
One example is https://kunde.comdirect.de/lp/wt/login:

image

### PR DESCRIPTION
**Pull request by perry-mitchell**
A release notes feature:

image

Closes #218 - Some inputs not recognised
Closes #163 - What's new section

### CODE REVIEW

File: source/setup/components/ReleaseNotes.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+        releaseNotes: PropTypes.string
+    };
+
+    componentDidMount() {
+        this.props.onReady();
+    }
+
+    render() {
+        return (
+            <Dialog title={`⚡️ What's New - v${__VERSION__}`}>
```
Comment (sallar) [inline source/setup/components/ReleaseNotes.js]: Can we move the condition higher so if there is no release notes it wouldn’t show an empty dialog?

File: source/setup/containers/ReleaseNotes.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    (state, ownProps) => ({
+        releaseNotes: getReleaseNotes(state)
+    }),
+    {
+        onReady: () => (dispatch, getState) => {
+            const releaseNotes = getReleaseNotes(getState());
+            if (!releaseNotes) {
+                fetchReleaseNotes();
+                setTimeout(() => {
+                    console.log("STATE", getState());
```
Comment (sallar) [inline source/setup/containers/ReleaseNotes.js]: Do we need the console log?

Comment (perry-mitchell) [reply source/setup/containers/ReleaseNotes.js]: Haha NOPE! 😄

File: source/setup/components/ReleaseNotes.js (hunk 1 above)
Comment (perry-mitchell) [reply source/setup/components/ReleaseNotes.js]: Agreed!

**Pull request merged**